config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   devise_for :users
+  get '/i/:token', to: 'wishlists_for_guests#show'
```

Comment:
^ because then we'll have methods like `invitations#show` that suggest we are showing invitations when in fact we are showing, well... wishlists, but for guests ;-)